src/main/java/diet/dietmanager/CommandLib.java

```diff
@@ -0,0 +1,30 @@
+package diet.dietmanager;
+
+import diet.dietmanager.command.*;
```

Comment:
Perhaps you can explicitly list out all the imports instead of using * to pass checkstyle